luminoso_api/client.py

```diff
@@ -234,14 +239,13 @@ def delete(self, path='', **params):
     # Useful abstractions
     def change_path(self, path):
         """
-        Return a new LuminosoClient for a subpath of this one.  Effectively, a
-        shortcut for instantiating a new LuminosoClient for that path.
+        Change the client's path for a subpath of this one.
```

Comment:
I honestly don't know why this docstring ever said "subpath of", given that you can give it a path that starts with `/` to get a path that isn't a subpath.  Definitely the word "subpath" should be taken out, and the docstring might want to be more explicit about the fact that you can use it to go to a subpath *or* to a new path entirely.

Reply:
Another note about this change: the `README.md` also needs to be updated, as it includes in its examples `project = client.change_path('/projects/' + project_id)`.  That technically works, but is very misleading about what it's doing.